UPGRADE-3.4.md

```diff
@@ -18,6 +18,11 @@ Finder
    deprecated and will be removed in 4.0 as it used to fix a bug which existed
    before version 5.5.23/5.6.7.
 
+Form
+----
+
+  * Deprecated `ChoiceLoaderInterface` implementation in `TimezoneType`.
```

Comment:
New options are just new features, so not required for any upgrade step. I tend to doc them in Form/CHANGELOG.